code/datums/elements/art.dm

```diff
@@ -49,7 +49,7 @@
 	var/mult = 1
 	if(isobj(source))
 		var/obj/art_piece = source
-		mult = art_piece.get_integrity() / art_piece.max_integrity
+		mult = max(art_piece.get_integrity() / art_piece.max_integrity, 1)// prevent cases where integrity is 0
```

Comment:
this just breaks this completely. it will never be above 1, this function is designed to go below 1

Reply:
:(